networking_bigswitch/bsnlldp/send_lldp.py

```diff
@@ -294,6 +294,106 @@ def daemonize():
         os._exit(os.EX_OK)
 
 
+PCI_IDS_DIR = "/run/pci_ids"
+# read lldp status for different interfaces
+lldp_status = {}
```

Comment:
since you define it globally and reuse it on every execution, you should probably do a `lldp_status.clear()` before you do a `read_lldp_status_from_device()`